packages/es-indexer/src/listener.ts

```diff
@@ -68,6 +70,10 @@ export async function startListener() {
       if (!row.play_item_id) return // when could this happen?
       pending.trackIds.add(row.play_item_id)
     },
+    aggregate_track: (row: AggregateTrackRow) => {
+      if (!row.track_id) return // when could this happen?
```

Comment:
i agree. when?

Reply:
this is a copy from above! i also don't know but im scared 🗡️

Reply:
lol yea i saw, but it's a good question

Reply:
@stereosteve any context?